lib/stat_tracker.rb

```diff
@@ -31,7 +30,8 @@ def self.from_csv(locations_params)
 
     game_teams_array = []
     CSV.foreach(game_teams_file, headers: true, header_converters: :symbol) do |row|
-      game_teams_array << row
+      game_team = GameTeam.new(row)
+      game_teams_array << game_team
```

Comment:
added GameTeam class to the stat_tracker class so now we're creating an array of game_team objects